RiotSharp/Endpoints/ClashEndpoint/Models/ClashTournamentPhase.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace RiotSharp.Endpoints.ClashEndpoint.Models
+{
+    /// <summary>
+    /// This model class defines properties of tournament phase model in clash
+    /// </summary>
+    public class ClashTournamentPhase
+    {
+        /// <summary>
+        /// Id of the tournament phase
+        /// </summary>
+        [JsonProperty("id")]
+        public int Id { get; set; }
+        
+        /// <summary>
+        /// registration start time in tournament phase in ms
+        /// </summary>
+        [JsonProperty("registrationTime")]
+        public long RegistrationTime { get; set; }
```

Comment:
Deserialize to DateTime using `RiotSharp.Misc.Converters.DateTimeConverterFromLong`